### PR TITLE
feat: enhanced the behaviour of Bookmark and Delete button

### LINKED ISSUE #1364: [Feat]: Improve bookmark button's behaviour
### Description : 
Currently, the website features a bookmark button that allows users to save links. However, upon clicking the bookmark button, there is **no visual indication or feedback** to confirm that the link has been successfully bookmarked. 
So I want to add a toast notification to provide users with immediate feedback when they click the bookmark button.

### Current Behavior:
- There is no notification or any form of feedback/acknowledgment to inform the user that the link has been bookmarked.

### Desired Behavior:
- Upon clicking the bookmark button, trigger a toast notification to provide visual feedback to the user.
- The toast notification will convey a clear message indicating that the link has been bookmarked successfully.
- The toast notification will appear briefly and fade out naturally, ensuring a smooth and non-intrusive user experience.

This feature will enhance the user experience.
 

### Add screenshots

Add screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by kirtiraj22**
## Related Issue

Closes #1364

## Description

I have added a toast notification. The toast notifications provide visual feedback to the user when they interact with these buttons.

- When a user clicks on the bookmark button, a success toast notification is displayed with the message "Bookmarked". The notification has a green color to indicate success.
- Similarly, when a user clicks on the delete button, an error toast notification is shown with the message "Deleted". The notification has a red color to indicate an error.
- Both toast notifications have a duration of 2 seconds and can be manually closed by clicking the cross icon present on the notification.

This enhancement improves the user experience by providing clear feedback when the user performs these actions. The toast notifications are visually appealing and help in better communication with the user.

Please review the changes and let me know if any further adjustments or improvements are needed.

## GIF : 
ezgif com-video-to-gif

## Checklist

- [X] My code adheres to the established style guidelines of the project.
- [ ] I have included comments in areas that may be difficult to understand.
- [X] My changes have not introduced any new warnings.
- [X] I have conducted a self-review of my code.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -54,25 +56,39 @@ function App() {
       link: bookmark.link,
       description: bookmark.description,
     }
+    toast.success("Bookmarked", {
```
Comment (VenomFate-619) [inline src/App.jsx:59]: move this function to some other place like utils.
make it easier to use it like 
```
toastSuccess("Bookmarked" )
```

the first msg parameter will be required and the second object will be optional,
its default might be 
```
{ autoClose: 2000,
      closeOnClick: true,
      pauseOnHover: true,
      draggable: true,
      theme: "colored",
      }
```

File: src/App.jsx [hunk 2]
```diff
@@ -54,25 +56,39 @@ function App() {
...
+      pauseOnHover: true,
+      draggable: true,
+      theme: "colored",
+    })
     setBookmarks([...bookmarks, newBookmark])
   }
 
   // Remove Bookmark
   function deleteres(product) {
+    toast.error("Deleted", {
```
Comment (VenomFate-619) [inline src/App.jsx:71]: same as above

**Comment by JasonDsouza212** [comment]
Add a screen recording in all media widths

**Comment by JasonDsouza212** [comment]
@kirtiraj22 revert all the changes made in pnpm-lock.yaml

**Comment by JasonDsouza212** [comment]
@kirtiraj22 if you won't communicate regarding this within today the Issue will be assigned to others

**Comment by kirtiraj22** [comment]
Hey @JasonDsouza212 

Apologies for the inconvenience caused. It seems that I didn't receive the notifications for the previous two replies on my Gmail. I'm sorry for the delay in communication.

**Comment by kirtiraj22** [comment]
I will update the changes by tonight. 
Just to clarify, should I make the code modifications and implement the changes in the utils as suggested by @VenomFate-619, or should I simply revert the pnpm-lock.yaml file?

**Comment by JasonDsouza212** [comment]
@kirtiraj22, please make both changes. Additionally, modify the notification colors to red - #D71313 and green - #A2FF86.

**Comment by kirtiraj22** [comment]
Hey @JasonDsouza212, I apologize for the inconvenience, but I have decided to close the PR as I won't be able to work on it. I have some urgent commitments and will be away for a week. Kindly consider assigning this task to someone else. Again, my apologies for the miscommunication.

**Pull request closed without merging**